qiime2/sdk/tests/test_plugin_manager.py

```diff
@@ -60,6 +60,42 @@ def test_importable_types(self):
 
     # TODO: add tests for type/directory/transformer registrations
 
+    def test_importable_formats(self):
+        _format = self.pm.importable_formats
+        exp = {'IntSequenceV2DirectoryFormat':
+               FormatRecord(format='qiime2.plugin.model.directory_format.'
```

Comment:
Can you have these format records store the actual format type objects instead of strings? I think the tests will start failing once the assertion below is updated to check values.